app/scripts/modules/kubernetes/src/v2/pipelines/stages/findArtifactsFromResource/findArtifactsFromResourceConfig.html

```diff
@@ -0,0 +1,31 @@
+<div class="form-horizontal">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+
+  <h4>Manifest</h4>
+  <div class="horizontal-rule"></div>
+  <kubernetes-manifest-selector selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
+
+  <h4>Settings</h4>
+  <div class="horizontal-rule"></div>
+  <expected-artifact-multi-selector command="ctrl.$scope.stage"
+                                    ids-field="expectedArtifactIds"
+                                    artifact-label="Artifacts To Bind"
+                                    expected-artifacts="ctrl.expectedArtifacts">
+  </expected-artifact-multi-selector>
+
+  <div class="container-fluid form-horizontal">
+    <div class="form-group">
+      <div class="col-md-3 sm-label-right">
+        Keep default artifacts
```

Comment:
Can you had a help label describing what this does? Here are some examples: https://github.com/spinnaker/deck/pull/4723/files

Reply:
Removed in the simplified version (perhaps to return in a modified form in the future).